### PR TITLE
Remove Semgrep CI as a separate Semgrep ecosystem

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -32,11 +32,10 @@ Its rules look like the code you already write; no abstract syntax trees, regex
...
 
-* [Semgrep](getting-started/) - the open-source command line tool at the heart of everything
-* [Semgrep CI](semgrep-ci/overview/) - a specialized Docker image for running Semgrep in CI environments
-* [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules
-* [Semgrep Registry](https://semgrep.dev/explore) - 2,000+ community-driven rules covering security, correctness, and performance bugs
-* [Semgrep App](https://semgrep.dev/manage) - deploy, manage, and monitor Semgrep at scale with free and paid tiers
+* [Semgrep](getting-started/) - The open-source command line tool at the heart of everything. You can run it also in [CI environment](semgrep-ci/overview/)!
+* [Semgrep Playground](https://semgrep.dev/editor) - An online interactive tool for writing and sharing rules.
+* [Semgrep Registry](https://semgrep.dev/explore) - 2,000+ community-driven rules covering security, correctness, and performance bugs.
+* [Semgrep App](https://semgrep.dev/manage) - Deploy, manage, and monitor Semgrep at scale with free and paid tiers.
```
Comment (pabloest) [inline docs/index.md:38]:
```suggestion
* [Semgrep App](https://semgrep.dev/manage) - Deploy, manage, and monitor Semgrep at scale with free and paid tiers. Integrates with CI providers such as GitHub, GitLab, CircleCI, and more.
```

File: docs/index.md [hunk 2]
```diff
@@ -32,11 +32,10 @@ Its rules look like the code you already write; no abstract syntax trees, regex
 
 The Semgrep ecosystem includes:
 
-* [Semgrep](getting-started/) - the open-source command line tool at the heart of everything
-* [Semgrep CI](semgrep-ci/overview/) - a specialized Docker image for running Semgrep in CI environments
-* [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules
-* [Semgrep Registry](https://semgrep.dev/explore) - 2,000+ community-driven rules covering security, correctness, and performance bugs
-* [Semgrep App](https://semgrep.dev/manage) - deploy, manage, and monitor Semgrep at scale with free and paid tiers
+* [Semgrep](getting-started/) - The open-source command line tool at the heart of everything. You can run it also in [CI environment](semgrep-ci/overview/)!
```
Comment (pabloest) [inline docs/index.md:35]:
```suggestion
* [Semgrep](getting-started/) - The open-source command line tool at the heart of everything.
```

(I propose moving the CI mention to the App bullet point, see the suggestion below)

**Comment by adamkvitek** [comment]
@pabloest agree with your review. Thank you!

**Review by pabloest — APPROVED** [review]

**Comment by pabloest** [comment]
Thanks, @adamkvitek !

Can you make a similar PR to update the equivalent section in th README file in the [Semgrep repo](https://github.com/returntocorp/semgrep)?

**Comment by adamkvitek** [comment]
@pabloest done - https://github.com/returntocorp/semgrep/pull/5793

**Pull request merged**